base/deprecated.jl

```diff
@@ -1076,4 +1077,6 @@ eval(Base.LinAlg, quote
     end
 end)
 
+Filesystem.stop_watching(stream::Filesystem._FDWatcher) = depwarn("stop_watching(::_FDWatcher) is removed")
```

Comment:
It should be removed from the code. I highly doubt anyone was using it though, since it was also incorrect to be calling it before this removed it.